src/cow-react/modules/application/containers/WithLDProvider.tsx

```diff
@@ -0,0 +1,13 @@
+import { LAUNCH_DARKLY_CLIENT_KEY } from '@cow/constants/launchDarkly'
+import { withLDProvider } from 'launchdarkly-react-client-sdk'
+import { PropsWithChildren } from 'react'
+
+function InnerWithLDProvider({ children }: PropsWithChildren) {
+  return <>{children}</>
+}
+export const WithLDProvider = withLDProvider<PropsWithChildren & JSX.IntrinsicAttributes>({
```

Comment:
This is not technically part of cowfortune. 

Also, I wouldn't abbreviate to LD, i had no idea first when I saw it.